### PR TITLE
Synapse 1.65

### PR DESCRIPTION
**Pull request by babolivier**
Preview: https://pr1404--matrix-org-previews.netlify.app

### CODE REVIEW

File: gatsby/content/blog/2022/08/2022-08-17-synapse-1.65.mdx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+Hey everyone! We've just released [Synapse
+1.65](https://github.com/matrix-org/synapse/releases/tag/v1.65.0)! Let's have a
+peek at what's inside.
+
+## Private read receipts
+
+A feature that the more privacy-focused users of Matrix have been missing was
+the ability to hide read receipts from other users. Read receipts in rooms can
+tell a user which messages another user has read in a room. However, they can
+also be unwelcomed indicator that a user is currently reading a certain room,
```
Comment (reivilibre) [inline gatsby/content/blog/2022/08/2022-08-17-synapse-1.65.mdx:18]:
```suggestion
also be an unwelcome indicator that a user is currently reading a certain room,
```

File: gatsby/content/blog/2022/08/2022-08-17-synapse-1.65.mdx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+Synapse 1.65 implements the new experimental error codes documented by
+[MSC3848](https://github.com/matrix-org/matrix-spec-proposals/pull/3827). Once
+stabilised, these error codes will allow clients to show more specific errors to
+their users about why an event could not be sent.
+
+This release of Synapse also stabilises the implementation of
+[MSC3827](https://github.com/matrix-org/matrix-spec-proposals/pull/3827), which
+allows filtering public room searches on room types. This means it is now
+possible to search specifically for public spaces.
```
Comment (reivilibre) [inline gatsby/content/blog/2022/08/2022-08-17-synapse-1.65.mdx:61]: this seems like quite big news TBH! Not sure it deserves to be left as a footnote, I think this is potentially more interesting than the module APIs to most people, at least to me

**Review by reivilibre — APPROVED** [review]
seems good otherwise :)

File: gatsby/content/blog/2022/08/2022-08-17-synapse-1.65.mdx (hunk 2 above)
Comment (babolivier) [reply gatsby/content/blog/2022/08/2022-08-17-synapse-1.65.mdx:61]: That's fair, though it was already mentioned in more details in [the 1.63 blog](https://matrix.org/blog/2022/07/20/synapse-1-63-released). I'll add a link to said post.

Comment (babolivier) [reply gatsby/content/blog/2022/08/2022-08-17-synapse-1.65.mdx]: (and I've bumped it to the top of "everything else")

**Pull request merged**